cpu/stm32f4/include/periph_cpu.h

```diff
@@ -49,7 +49,7 @@ enum {
  * @brief   Available number of ADC devices
  */
 #if defined(CPU_MODEL_STM32F401RE) || defined(CPU_MODEL_STM32F410RB) \
-    || defined(CPU_MODEL_STM32F411RE) \
+    || defined(CPU_MODEL_STM32F411RE) || defined(CPU_MODEL_STM32F412ZG) \
```

Comment:
Looks unrelated, is this a pending bug ?

Reply:
Not sure, I'm think I messed up when rebasing. I'll take a look.